### PR TITLE
chroot: added an example

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Added a simple example of chroot connection plugin

Fixes: #6365

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
plugins/connection/chroot.py

### CODE REVIEW

File: plugins/connection/chroot.py [hunk 1]
```diff
@@ -48,6 +48,25 @@
...
+# [chroots]
+# /path/to/debootstrap
+# /path/to/feboostrap
+# /path/to/lxc-image
+# /path/to/chroot
+
+# playbook
+---
+- hosts: chroots
+  connection: chroot
```
Comment (felixfontein) [inline plugins/connection/chroot.py:63]:
```suggestion
  connection: community.general.chroot
```

File: plugins/connection/chroot.py [hunk 2]
```diff
@@ -48,6 +48,25 @@
...
+  tasks:
+    - debug:
+        msg: "This is coming from Chroot environment"
```
Comment (felixfontein) [inline plugins/connection/chroot.py:66]:
```suggestion
        msg: "This is coming from chroot environment"
```

**Review by felixfontein — COMMENTED** [review]
Thanks for adding this example!

**Review by russoz — APPROVED** [review]
LGTM (after the requested changes)

**Review by russoz — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
@Akasurde @russoz thanks!

**Pull request merged**